doc/conf.py

```diff
@@ -682,6 +682,22 @@ def notebook_modification_function(notebook_content, notebook_filename):
     # imports inside functions
     code_lines.extend(["import matplotlib", "import pandas"])
 
+    # Work around https://github.com/jupyterlite/pyodide-kernel/issues/166
+    # and https://github.com/pyodide/micropip/issues/223 by installing the
+    # dependencies first, and then scikit-learn from Anaconda.org.
+    if "dev" in release:
+        dev_docs_specific_code = [
+            "import micropip",
+            "await micropip.install(['joblib', 'threadpoolctl', 'scipy'])",
+            "await micropip.install(\n"
+            "   'scikit-learn',\n"
+            "   index_urls='https://pypi.anaconda.org/scientific-python-nightly-wheels/simple',\n"
+            "   pre=True,\n"
```

Comment:
Maybe something like this would be better?
```suggestion
            f"   'scikit-learn={release!r}',\n"
            "   index_urls='https://pypi.anaconda.org/scientific-python-nightly-wheels/simple',\n"
```

There may be a future where we have a 1.6.1 Pyodide wheel and this will work more easily?

Reply:
> Maybe something like this would be better?

I agree!

> There may be a future where we have a 1.6.1 Pyodide wheel and this will work more easily?

There is [`jupyterlite-pyodide-kernel==0.6.0a4`](https://github.com/jupyterlite/pyodide-kernel/releases/tag/v0.6.0a4) which uses Pyodide 0.27.4 with scikit-learn 1.6.1, should I switch to it or would you like to wait until 0.6.0 stable lands? Here's the linked issue: https://github.com/jupyterlite/jupyterlite/issues/1554

Reply:
> There is [jupyterlite-pyodide-kernel==0.6.0a4](https://github.com/jupyterlite/pyodide-kernel/releases/tag/v0.6.0a4) which uses Pyodide 0.27.4 with scikit-learn 1.6.1, should I switch to it or would you like to wait until 0.6.0 stable lands?

Thanks! I think we are fine to wait until jupyterlite-pyodide-kernel 0.6.0 lands.